uri/exid-types/BillionGraves-CemeteryId.yaml

```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+label: BillionGraves Cemetery ID
+
+lang: en
+
+type: uri
+
+uri: https://www.billiongraves.com/cemetery/name/
```

Comment:
I noticed in exid-types.json that some uris end in `/` and some don't.  Shouldn't they be consistent?

The spec says: "If the authority maintains stable URLs for each identifier it issues, it is recommended that the [TYPE](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#TYPE) payload be selected such that appending the [EXID](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#EXID) payload to it yields that URL."

If that is taken literally, then for everything that follows this recommendation, shouldn't the uri end in `/` like this one does?  I.e., is the URI for FamilySearch Memory ID, etc. wrong?